### ISSUE TITLE
Improve signature verification formatting

### ISSUE THREAD

**Issue by nodech** [braydonf/gpk#20]
Even though gpg output is OKAY to read, would be nice to have better formatting, something like:
```
gpg: [✓] [Fri May  3 03:47:53 2019 CEST] using 5B7D C58D 90FE C1E9 90A3 10BA F24F 232D 108B 3AD4 from Braydon Fuller <braydon@purse.io>.
gpg: [✕] WARNING: Not certified with trusted signature.
```

Or something with colors. This is just a minor thing, but maybe we could have it ? )

**Comment by braydonf** [comment]
Possibly, currently the output is piped directly from `git` and `gpg`.

**Comment by nodech** [comment]
yep, ALSO for github packages that DO NOT have signatures, have a big error message and exit the process or maybe skip and LOG errors in the end alerting users about not signed packages.
